.github/workflows/CD.yml

```diff
@@ -590,13 +596,17 @@ jobs:
 
       - name: Pull image from integration test ECR, tag with input version and push to DockerHub
         uses: akhilerm/tag-push-action@v2.0.0
-        if: steps.release-latest-image.outputs.cache-hit != 'true'
+        if: steps.release-latest-image-s3.outputs.cache-hit != 'true'
         with:
           src: public.ecr.aws/${{ env.ECR_REPO }}:${{ needs.release-checking.outputs.version }}
           dst: ${{ env.IMAGE_NAMESPACE }}/${{ env.IMAGE_NAME }}:${{ needs.release-checking.outputs.version }}
 
+      - name: Release binaries to s3 with latest version
```

Comment:
I would keep this separate.

Reply:
Reasonable. Separate the release-latest s3 and release-latest image into two different jobs.